tests/e2e/specs/dashboard-samples/Documentation.spec.ts

```diff
@@ -0,0 +1,132 @@
+		CLASSES.KubernetesCommandLineToolsExecutor
+	);
+	kubernetesCommandLineToolsExecutor.workspaceName = workspaceName;
+	const loginTests: LoginTests = e2eContainer.get(CLASSES.LoginTests);
+	const dashboard: Dashboard = e2eContainer.get(CLASSES.Dashboard);
+	const workspaces: Workspaces = e2eContainer.get(CLASSES.Workspaces);
+	const workspaceDetails: WorkspaceDetails = e2eContainer.get(CLASSES.WorkspaceDetails);
+	const shellExecutor: ShellExecutor = e2eContainer.get(CLASSES.ShellExecutor);
+	const browserTabsUtil: BrowserTabsUtil = e2eContainer.get(CLASSES.BrowserTabsUtil);
+
+	const testingVersion: string = BASE_TEST_CONSTANTS.TESTING_APPLICATION_VERSION;
+	let parentGUID: string = '';
+
+	// get links from product version github branch
+	const { docs, links, productVersion }: any = JSON.parse(
+		shellExecutor.curl(
+			`https://raw.githubusercontent.com/redhat-developer/devspaces-images/devspaces-${testingVersion}-rhel-8/devspaces-dashboard/packages/dashboard-frontend/assets/branding/product.json`
+		)
+	);
+	const { webSocketTroubleshooting, workspace, devfile, general, storageTypes } = docs;
+
+	suiteSetup('Login into OC client and apply default DevFile', function (): void {
+		kubernetesCommandLineToolsExecutor.loginToOcp();
+		kubernetesCommandLineToolsExecutor.applyYamlConfigurationAsFile(pathToSampleFile);
+		shellExecutor.wait(5);
+	});
+
+	suiteTeardown('Delete default DevWorkspace', function (): void {
+		kubernetesCommandLineToolsExecutor.deleteDevWorkspace();
+	});
+
+	test('Check if product.json config contains correct application version', function (): void {
+		[productVersion, links[1].href, devfile, workspace, general, storageTypes, webSocketTroubleshooting].forEach((e): void => {
+			expect(e).contains(testingVersion);
+		});
+	});
+
+	loginTests.loginIntoChe();
+
+	test('Check if documentation section "About" present on Dashboard', async function (): Promise<void> {
+		await dashboard.openAboutMenu();
+	});
+
+	test('Check if documentation section "About" in top menu has required links', async function (): Promise<void> {
+		parentGUID = await browserTabsUtil.getCurrentWindowHandle();
+		for (const link of links) {
+			await dashboard.selectAboutMenuItem(link.text);
+			await browserTabsUtil.waitAndSwitchToAnotherWindow(parentGUID);
+			const currentUrl: string = await browserTabsUtil.getCurrentUrl();
+			expect(link.href).oneOf([currentUrl, currentUrl + '/']);
+			await browserTabsUtil.switchToWindow(parentGUID);
+			await browserTabsUtil.closeAllTabsExceptCurrent();
+		}
+	});
+
+	test('Check if "About" dialog menu contains correct application version', async function (): Promise<void> {
+		await dashboard.selectAboutMenuItem('About');
+		for (const e of dashboard.getAboutDialogWindowMenuElements()) {
+			expect(await dashboard.driverHelper.waitVisibilityBoolean(e)).is.true;
+		}
+		expect(await dashboard.getApplicationVersionFromAboutDialogWindow()).and.eqls(productVersion);
+		await dashboard.closeAboutDialogWindow();
```

Comment:
Could you please add verification of Username to the test, following test scenario from https://gitlab.cee.redhat.com/codeready-workspaces/test-cases/-/blob/master/tests/functional/miscellaneous/m19-check-links-to-documentation-page-in-dashboard.md ?

Reply:
thanks